pymc3/model.py

```diff
@@ -1122,7 +1122,11 @@ def add_random_variable(self, var, dims=None):
 
     @property
     def prefix(self):
-        return "%s_" % self.name if self.name else ""
+        if self.name:
+            val = self.name
+            return f"{val}_"
+        else: 
+            return ""
```

Comment:
```suggestion
        return f"{self.name}_" if self.name else ""
```

Reply:
Oh is it? OK maybe I didnt understand it well enough